spell/spell-crafter-goerli-workflow.md

```diff
@@ -127,15 +127,13 @@ PR: https://github.com/makerdao/spells-goerli/pull/TODO
   * [ ] `deployed_spell_block: <block number>`
     * [ ] Run `make deploy-info tx=<tx_hash>` to get timestamp and block number
 * [ ] Run Tests Locally with deployed spell address
+* [ ] Archive spell via `date="YYYY-MM-DD" make archive-spell` (date as per the exec doc)
```

Comment:
SO:

For goerli in the past we have been sometimes going off the goerli **deployment** date and sometimes going off the goerli cast date.

Let's say a spell is scheduled for mainnet deploy and handoff to GovAlpha on the 17th, but Goerli spell is deployed and cast on the 15th or 16th.

Do we want archive for Goerli to be labelled 15/16 (deploy and cast date) or 17 (mainnet "exec doc" date)?

Discussed with review team a week or two ago and this was contentious and we have never used one strategy uniformly, so this is up to you to dictate now (or we can seek team consensus).

Reply:
18cf91c